Documentation/ApiOverview/PasswordPolicies/Index.rst

```diff
@@ -155,6 +155,10 @@ equal to the old password. The validator must always be configured with
 the exclude action :php:`\TYPO3\CMS\Core\PasswordPolicy\PasswordPolicyAction::NEW_USER_PASSWORD`,
 because it should be excluded, when a new user account is created.
 
+Third-party validators
+----------------------
+
+The extension :t3ext:`add_pwd_policy` provides additional validators.
```

Comment:
```suggestion
The extension :composer:`derhansen/add_pwd_policy` provides additional validators.
```

I would suggest to link to the extension using the newly introduces composer role as this enables user two copy the composer require code with just two clicks or copy the composer name directly to install it: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/Reference/ReStructuredText/Links/Composer.html

Reply:
Now the link to TER and therefore the documentation is missing. I prefer using t3ext for now, as all information is on the extension page (including composer package name).